### ISSUE TITLE
Not able to sort by nullity

### ISSUE THREAD

**Issue by Hlaboa** [ResidentMario/missingno#86]
Neither with `msno.nullity_sort` nor with `msno.bar`, I'm not able to sort the columns by nullity.
In both functions I try with `sort='ascending'`.  It doesn't throw me errors, but the plot doesn't change.

**Comment by ResidentMario** [comment]
Nullity sort is working, actually; but I think you're misinterpreting what it's supposed to do.

`msno.nullity_sort` resorts data based on the number of null observations in the column. In other words, it is a row-oriented nullity sort. The `ascending` sort does this in the forward direction:

```
print(
    msno.nullity_sort(pd.DataFrame({'A': [1, 2, 3, 4], 'B': [np.nan, 2, 3, 4]}),
    sort='ascending')
)

   A    B
0  1  NaN
1  2  2.0
2  3  3.0
3  4  4.0
```

`descending` sort does this in the reverse direction:

```
print(msno.nullity_sort(
    pd.DataFrame({'A': [1, 2, 3, 4], 'B': [np.nan, 2, 3, 4]}), sort='descending')
)

   A    B
3  4  4.0
2  3  3.0
1  2  2.0
0  1  NaN
```

While sorting the data has an important and highly visible effect on `msno.matrix`, it does nothing when applied to data passed to `msno.bar`. The only value that matters in `msno.bar` is the per-column nullity count, which is unaffected by the data sort order (or lack thereof).

However there is another form of sorting that makes sense in the `msno.bar` column, which is sorting on the per-column nullity (which I think is what you are asking for). It wouldn't be too hard to implement this, and it would certainly be better than the current `sort` parameter on `msno.bar`, which, well, doesn't do anything.

**Comment by ResidentMario** [comment]
Addressed in #87.

**Comment by Hlaboa** [comment]
yes, you were right. I misinterpreted what it supposed to do. Thanks for #87 update!
